test/tools/jsontests/StateTests.cpp

```diff
@@ -104,10 +104,6 @@ class generaltestfixture
 	generaltestfixture()
 	{
 		string casename = boost::unit_test::framework::current_test_case().p_name;
-		if (casename == "stBoundsTest" && !test::Options::get().memory)
-			return;
-		if (casename == "stMemoryStressTest" && !test::Options::get().memory)
-			return;
```

Comment:
stMemoryStress test will run even if memory flag is not defined 
I think that with gaslimit of 250M travis could do that